### DISCUSSION TITLE
The connection could not be made due to the target computer's positive rejection. (localhost:1080)

### DISCUSSION THREAD

**Discussion by Rain-0x01-39** [ppy/osu#28432] [Q&A]
Unable to log in
[compressed-logs.zip](https://github.com/user-attachments/files/15748558/compressed-logs.zip)

**Comment by peppy** [comment]
sounds like you might has a proxy setup that doesnt' work with osu!.

**Comment by Rain-0x01-39** [comment]
I think osu! should have proxy settings

**Reply by BlockHead007** [reply]
#25008
